django_ses_plus/models.py

```diff
@@ -50,11 +71,23 @@ def send_email(self, subject, template_path, context, from_email=None, language=
         if language:
             translation.activate(language)
 
+        if attachments is not None:
+            assert isinstance(attachments, list), "Attachments should be a `list` of `dict` objects."
+
+            for attachment in attachments:
+                assert all([key in attachment for key in ["name", "content", "type"]]), "Attachments should contain `name`, `content` and `type`."
+
+                if isinstance(attachment["content"], bytes):
+                    attachment["content"] = base64.b64encode(attachment["content"]).decode("utf-8")
```

Comment:
Could you please add comment that stating this decode is done to make it json serializable which is default serializer for celery.

Reply:
Can we imagine a better way than sending the file to Task Queue? Imagine someone is attaching a file around 20mb. It will add about 30 mb of "task info" to the task queue.

Reply:
The maximum allowed attachment size is 10 MB for Amazon SES. We can put a max limit here or `utils.send_email`. Other than that a solution could be first creating `SentEmailAttachment` objects with null `sent_email` field, passing ids of these newly created objects and then getting them in the task.

Reply:
> Can we imagine a better way than sending the file to Task Queue? Imagine someone is attaching a file around 20mb. It will add about 30 mb of "task info" to the task queue.

What kind of problem can this bring?